src/test/java/com/training/services/tests/VoyagesTest.java

```diff
@@ -73,14 +77,15 @@ public void testVoyageSailDurationValidate() {
     @Test
     public void testVoyageShipCodeValidate() {
         Response voyageResponse = getResponse();
-        logger.info("status->" + voyageResponse.getStatus());
-        logger.info("totalVoyagesCount->" + voyageResponse.getPayload().getVoyages().size());
+
         Assertions.assertThat(voyageResponse.getStatus()).isEqualTo("200")
                 .as(" status is not 200");
+
         List<Voyages> voyages = voyageResponse.getPayload().getVoyages();
-        for (int i = 0; i < voyages.size(); i++) {
-            softAssert.assertThat(voyages.get(i).getShipCode()).isEqualTo("AL");
-        }
+
+        IntStream.range(0, voyages.size()).forEach(index -> {
+            softAssert.assertThat(voyages.get(index).getShipCode()).isEqualTo("AL");
+        });
     }
```

Comment:
Do we really need index in this?
Why cant we use - response.getPayload().getVoyages().forEach()?